### PR TITLE
Split workflows

### PR DESCRIPTION
**Pull request by cosmo0920**
Using if condition branches is a bit messy work flow.
Splitting linux workflow into apt and yum should be simple.

### CODE REVIEW

File: .github/workflows/apt.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Linux
```
Comment (ashie) [inline .github/workflows/apt.yml:1]: Pls set appropriate name.

Comment (cosmo0920) [reply .github/workflows/apt.yml]: I'd used Apt based Linux and Yum based Linux.

Comment (kenhys) [reply .github/workflows/apt.yml]: Debian and derivatives?

Comment (kenhys) [reply .github/workflows/apt.yml]: except for `name:`, LGTM.

**Comment by cosmo0920** [comment]
I'll merge this PR after Amazon Linux 2 package is built.

**Comment by cosmo0920** [comment]
Amazon Linux 2 rpm package is built successfully:
https://github.com/fluent-plugins-nursery/td-agent-builder/actions/runs/136689355

**Pull request merged**